src/plugins/expressions/common/expression_functions/arguments.ts

```diff
@@ -59,7 +60,7 @@ interface BaseArgumentType<T> {
    */
   deprecated?: boolean;
   /** Help text for the Argument to be displayed in the Expression Editor */
-  help: string;
+  help?: string;
```

Comment:
I found a lot of instances where the `help` option was either left empty or forced as a string. I think this is an unnecessary property, which was already defaulting to `''` when not defined.